main/terminal.cc

```diff
@@ -2806,11 +2753,11 @@ const genericChar* Terminal::ReplaceSymbols(const genericChar* str)
                     GetMacAddress(tmp, STRLENGTH);
                     break;
                 case 6:  // machinekey
-                    GetMachineDigest(tmp, 20);
+                    GetUnameInfo(tmp, 20);
```

Comment:
GetMachineDigest is not used anywhere else. I replaced it with uname info, but I'm not sure what else to put here